src/deluge/gui/ui/browser/browser.cpp

```diff
@@ -746,7 +746,7 @@ Error Browser::arrivedInNewFolder(int32_t direction, char const* filenameToStart
 					goto tryAgain;
 				}
 				numberStartPos = endSearchString.getLength() + 1;
-				error = endSearchString.concatenate(display->haveOLED() ? " :" : "_:");
+				error = endSearchString.concatenate(" :");
```

Comment:
there's a bunch of code right above this that assumes there's an underscore before the incremented number so I think this will cause more bugs